### PR TITLE
Publish to Windows Package Managar (WinGet)

### PR DESCRIPTION
**Pull request by vedantmgoyal2009**
This action automatically generates manifests for WinGet Community Repository ([microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs)) and submits them.

Before merging this:
1. Fork [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs) under @oliverschwendener.

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
@oliverschwendener can this be merged? you only need to fork microsoft/winget-pkgs and the work is done.

**Comment by vedantmgoyal2009** [comment]
@oliverschwendener can this be merged? you only need to fork microsoft/winget-pkgs and the work is done.

File: .github/workflows/winget.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  release:
+    types: [released]
+jobs:
+  publish:
+    runs-on: windows-latest # action can only be run on windows
+    steps:
+      - uses: vedantmgoyal2009/winget-releaser@latest
+        with:
+          identifier: OliverSchwendener.ueli
+          token: ${{ secrets.GH_TOKEN }}
```
Comment (oliverschwendener) [inline .github/workflows/winget.yml:12]: Can you explain why this environment variable is needed?

Comment (vedantmgoyal2009) [reply .github/workflows/winget.yml:12]: To create pull requests at https://github.com/microsoft/winget-pkgs.

**Comment by vedantmgoyal2009** [comment]
@oliverschwendener To create pull requests at https://github.com/microsoft/winget-pkgs.

**Comment by oliverschwendener** [comment]
Anyone with this token has full access to all of my private repositories. I don't think that is needed to create pull requests in another repo. I think I have to create a separate access token for this use case.

**Comment by oliverschwendener** [comment]
Do you know what is needed for creating PRs? Here are all the options:

image

**Comment by vedantmgoyal2009** [comment]
Only `public_repo` will work. `workflow` is only required in some special cases, like: https://github.com/sandboxie-plus/Sandboxie/runs/7931748039?check_suite_focus=true#step:2:173.

**Comment by vedantmgoyal2009** [comment]
@oliverschwendener Only `public_repo` will work. `workflow` is only required in some special cases, like: https://github.com/sandboxie-plus/Sandboxie/runs/7931748039?check_suite_focus=true#step:2:173.

**Comment by vedantmgoyal2009** [comment]
@oliverschwendener hello...

**Comment by vedantmgoyal2009** [comment]
@oliverschwendener any updates?

**Comment by oliverschwendener** [comment]
Sorry for my inactivity here, I'm currently struggling to find time for this project. I now forked the `winget-pkgs` repository. Please use the secret `WIN_GET_REGISTRY` instead of `GH_TOKEN`.

**Review by oliverschwendener — CHANGES_REQUESTED** [review]
See my comment above.

**Comment by oliverschwendener** [comment]
@vedantmgoyal2009 So how does this work? On every new release, it will publish the new release to the WinGet registry? Is there anything that needs to be done manually when a future version is released?

**Comment by vedantmgoyal2009** [comment]
On every release made, the action will create a PR at microsoft/winget-pkgs to add manifests to WinGet. The PRs will be created using the token you've supplied. 

I hope you've given completed the steps mentioned at https://github.com/vedantmgoyal2009/winget-releaser/#getting-started-

**Pull request merged**